GPT/lists/modelInsertionModifier.talon-list

```diff
@@ -0,0 +1,17 @@
+list:  user.modelInsertionModifier
+
+-
+
+# A list of adjectives/adverbs describing how the result of the GPT query should be returned
+
+# Inserted as a snippet
+snip: snip
+
+# paste in place
```

Comment:
I was thinking of it as kind of like the null modifier-as in: insert this where it is without changing the cursor at all. As this is currently written you cannot combine destinations with modifiers. All the destinations are targets outside of the editor. I don't think that it makes sense to paste a snippet to browser or clipboard for example.

Reply:
`to this` follows the pattern of the cursorless grammar. I would prefer to follow that convention. 

It may not feel like a lot of complexity but I am really trying to focus on getting users using anything beyond just the default destination/source. It makes the documentation site a bit nicer if we keep the `source` `destination` pattern.

I agree that it doesn't make much sense to apply `snip` to the clipboard, but it follows the grammar and doesn't cause anything to break. Once we starting allowing multiple modifiers which are mutually exclusive with a destination, it starts to get messy.

Reply:
If you'd prefer I structures so that you can apply insertion modifiers to every destination then I can readjust. That is preferable to me. 👍